features/provider/admin/messages/outbox.feature

```diff
@@ -0,0 +1,27 @@
+@javascript
+Feature: Outbox messages
+  In order to facilitate communication between me and my buyers
+  As a provider
+  I want to have an internal messaging system at my disposal
+
+  Background:
+    Given a provider "foo.3scale.localhost"
+    Given a default application plan "Basic" of provider "foo.3scale.localhost"
+      And provider "foo.3scale.localhost" has multiple applications enabled
+
+    Given a following buyers with applications exists:
+      | name | provider        | applications        |
+      | bob  | foo.3scale.localhost | BobApp              |
+      | jane | foo.3scale.localhost | JaneApp, JaneAppTwo |
+
+    Given current domain is the admin domain of provider "foo.3scale.localhost"
+      And I don't care about application keys
```

Comment:
Applications is a different scope and I don't thing it is related to outbox messages. All these steps can probably go away.
```suggestion
    Given a provider is logged in
```